politeiawww/www.go

```diff
@@ -440,6 +441,17 @@ func _main() error {
 		p.setPoliteiaWWWRoutes()
 	case cmsWWWMode:
 		p.setCMSWWWRoutes()
+		cockroachdb.UseLogger(cockroachdbLog)
```

Comment:
This is referencing the cache.  I think you mean:

```suggestion
	cmsdb.UseLogger(cockroachdbLog)
```